csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs

```diff
@@ -1,28 +1,29 @@
-using Deprecation;
-
+using Deprecation.StringUtils;
 namespace DeprecationTests
```

Comment:
You can take a look at the other comment regarding this to see what is meant by file scoped namesapce